opensrp-anc/src/main/java/org/smartregister/anc/library/util/ConfigHelperUtils.java

```diff
@@ -9,7 +9,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ConfigHelper {
+public class ConfigHelperUtils {
```

Comment:
I thought the decision was to not change the Util names but relax the rules?

Reply:
This was just for the old libs like core, child and immunization
New libs would have to conform